cartridges/int_cloudinary/cartridge/scripts/jobs/steps/cldUpdateProductImagesForFeeds.js

```diff
@@ -84,7 +84,7 @@ function writeProductFileContent(cloudinaryUrlStreamWriter, productSearchHitsItr
             if (!empty(imgAssetsSorted) && !empty(imgAssetsSorted.resources) && imgAssetsSorted.resources.length > 0) {
                 if (!empty(params.jobLastExecutionTime)) {
                     skipCurrentProduct = true;
-                    for (var i = 0; i < imgAssetsSorted.resources.length; i++) {
+                    for (let assetsSorted of imgAssetsSorted.resources) {
```

Comment:
Please use const

Reply:
As far as I know, we should use "let" in loops, the reason is the scope of the variable will be limited within the "for" block.

Reply:
'let' should be used for variables we intend to change after defining.
'const' should be used for variables we will never change after defining.
Because we redefine the same variable and never change it, we should use 'const'

Reply:
We got syntax error in job when I use const instead of let in for-off loop

Reply:
can you share the error?

Reply:
![image (17)](https://github.com/user-attachments/assets/81963325-4af6-410c-805c-4ef1b3713a16)

Reply:
![image](https://github.com/user-attachments/assets/e72e8331-34f7-4fc3-a93d-d1f8c3ac705f)

Reply:
thanks, maybe an old ES version issue